pymchelper/writers/trip98.py

```diff
@@ -1,8 +1,9 @@
 import time
 import logging
 import os
-
 import numpy as np
+from pymchelper import __version__ as _pmcversion
+from pytrip import __version__ as _ptversion
```

Comment:
The problem is that on purpose `pytrip` is not on the list of requirements for pymchelper.
Support of pytrip related functions is for "advanced users" who will install pytrip by themselves.
It was done like this for two reasons:
 - most of pymchelper users work with simple Fluka/SH12A simulation and don't need such heavy dependency
 - installation of pytrip under Windows and old Linuxes is problematic - it depends on many packages (scipy/paramiko etc)

I'd reccommend to do it in following way:
 - make an pytrip import and if it fails, then write a message to the user: "please install pytrip with `pip install pytrip98`"
 - if it succeeds then set the version to correct one
 - if it fails, then stop further execution